execution_workflows/IsoSCM/README.md

```diff
@@ -0,0 +1,76 @@
+# IsoSCM execution workflow
+
+===== intro =====
+
+Isoform Structural Change Model (IsoSCM) is a method for transcript assembly that incorporates change-point analysis to improve the 3' UTR annotation process.
+
+- Paper: [IsoSCM: improved and alternative 3′ UTR annotation using multiple change-point inference](https://rnajournal.cshlp.org/content/21/1/14)
+- GitHub: [shenkers/isoscm](https://github.com/shenkers/isoscm)
+
+
+## Remapping the testing data
```

Comment:
I think it would be nice to include this step of converting the provided input data into the input data that is suitable for the tool in the workflow itself since this step seems to be specific for isoscm. This way minimal preprocessing work outside of the workflow has to be done and the inputs for all the different workflows for benchmarking could be the same :) @dominikburri  what do you think?